tests/appsec/test_processor.py

```diff
@@ -107,3 +107,24 @@ def test_appsec_span_tags_snapshot(tracer):
         set_http_meta(span, {}, raw_uri="http://example.com/.git", status_code="404")
 
     assert "triggers" in json.loads(span.get_tag("_dd.appsec.json"))
+
+
+def test_appsec_span_rate_limit(tracer):
+    os.environ["DD_APPSEC_TRACE_RATE_LIMIT"] = "1"
```

Comment:
we have a helper `tests.utils.override_env` that can help here, especially for the `del` at the end.

e.g.

```python
with override_env({"DD_APPSEC_TRACE_RATE_LIMIT": "1"}):
    _enable_appsec(tracer)

with tracer.trace():
    ...
```

Reply:
🚀  sounds good!